std/uni.d

```diff
@@ -5134,6 +5140,708 @@ public auto utfMatcher(Char, Set)(Set set) @trusted
         static assert(false, "Only character types 'char' and 'wchar' are allowed");
 }
 
+// helper for assert/debug prints
+string utfWordAsBinary()(uint val, size_t sz=0)
+{
+    import std.algorithm : reverse;
+    import core.bitop;
+    import std.format;
+    char[] res;
+    if(sz == 0)
+        sz = (bsr(val)+8)/8;
+    while(sz){
+        res ~= format("%x", val & 0xFF);
```

Comment:
The function is named `utfWordAsBinary`, but "%x" is hex formatting. Should this be called `utfWordAsHex`?